src/tools/analyzer/modules/request.py

```diff
@@ -148,36 +148,57 @@ def consume_line(self, line, source, consume):
                 print(line)
         return found_results
 
-    def print_formatted_verbose(self, source, patterns):
+    def print_formatted_verbose(self, source):
         """
-        Parse line and print formatted verbose list_requests output
+        Parse log file and print formatted verbose list_requests output
 
         Args:
             source (Reader): source Reader object
-            patterns (list): List of regex patterns to use for
-                matching lines
         """
-        # Get CID number, and print the basic line first
-        for line in self.matched_line(source, patterns):
-            cid = self.print_formatted(line)
-
-            # Loop through each line with this CID number to extract and
-            # print the verbose data needed
-            verbose_patterns = ["(cache_req_send|cache_req_process_input|"
-                                "cache_req_search_send)"]
-            for cidline in self.matched_line(source, verbose_patterns):
+        data = {}
+        # collect cid log lines from single run through of parsing the log
+        # into dictionary # (cid, ts) -> logline_output
+        for line in source:
+            if "CID#" not in line:
+                continue
+
+            # parse CID and ts from line, key is a tuple of (cid,ts)
+            fields = line.split("[")
+            # timestamp to the minute, cut off seconds
+            ts = fields[0][:-6]
+            result = re.search('CID#[0-9]*', fields[3])
+            cid = result.group(0)
+
+            # if mapping exists, append line to output. Otherwise create new mapping
+            if (cid, ts) in data.keys():
+                data[cid, ts] += line
```

Comment:
Shouldn't this be `(cid, ts)` (like below)?

Reply:
Fixed, thanks.